common/tools/dev-tool/src/commands/admin/migrate-package.ts

```diff
@@ -352,18 +356,18 @@ async function upgradeTypeScriptConfig(tsconfigPath: string): Promise<void> {
   tsConfig.compilerOptions.moduleResolution = "NodeNext";
   tsConfig.compilerOptions.rootDir = ".";
   tsConfig.include = [
-    "./src/**/*.ts",
-    "./src/**/*.mts",
-    "./src/**/*.cts",
-    "./samples-dev/**/*.ts", // TODO: Check if samples-dev is needed
-    "./test/**/*.ts",
+    "src/**/*.ts",
```

Comment:
[Specifies an array of filenames or patterns to include in the program. These filenames are resolved relative to the directory containing the tsconfig.json file.](https://www.typescriptlang.org/tsconfig/#include)

Also makes https://github.com/Azure/azure-sdk-for-js/blob/main/common/tools/eslint-plugin-azure-sdk/docs/rules/ts-config-include.md happy